src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -66,5 +69,45 @@ public int Main(string[] args)
 
             return app.Execute(args);
         }
+
+        [Conditional("DEBUG")]
+        // TODO(anurse): This seems like a broadly useful helper
+        private static void WaitForOrLaunchDebuggerIfRequested(ref string[] args)
```

Comment:
This ended up being really useful. DNU "needs" it's own debug helper code because it's difficult to use the dnx one since you generally run dnu through a batch file so it's hard to insert arguments between "dnx" and "Microsoft.Framework.PackageManager". It's possible, but hard. Also, I did add a few extra features to the debug helper.

Happy to remove it if people really don't like it, but I think it's useful. Might even be a useful helper to put in a Sources package